docs/modules/ROOT/pages/documentation/payara-micro/asadmin/pre-and-post-boot-scripts.adoc

```diff
@@ -18,17 +16,18 @@ No line breaks are allowed.
 * Each _asadmin_ command must be formatted on the syntax `subcommand [options] [operands]`,
 as if you were running the commands in a multimode session from the command line.
 * Empty lines and comment lines (starting with the `#` character) are ignored.
-* Each asadmin command may contain variable references (e.g. to environment variables or system properties) as described in xref:/documentation/payara-server/server-configuration/var-substitution/usage-of-variables.adoc[Using variables]
+* Each _asadmin_ command may contain variable references (e.g. to environment variables or system properties) as described in xref:/documentation/payara-server/server-configuration/var-substitution/usage-of-variables.adoc[Using variables]
  (_Since Payara Micro 4.1.2.174_)
+* _asadmin_ commands that contain spaces in the value should be surrounded with quotations marks.
 
 Here's a sample of a post boot script used to configure an instance for production uses:
 
 [source, shell]
 ----
 #Disable dynamic reloading of applications
-set configs.config.server-config.admin-service.das-config.dynamic-reload-enabled=false
+set "configs.config.server-config.admin-service.das-config.dynamic-reload-enabled=false"
```

Comment:
Quotation marks aren't needed in any of these examples so if we add them to the examples, it might confuse people. Let's create a new example for setting a system property, e.g.

```suggestion
set configs.config.server-config.admin-service.das-config.dynamic-reload-enabled=false

```
# Set a system property which contains spaces
create-system-properties "property.name=This is a property value"
```
```